### PR TITLE
[4.0] Move status message below field

### PR DESCRIPTION
**Pull request by Quy**
### Summary of Changes
- Move message to the bottom of the field to prevent field shrinking and expanding when displaying the message
- Replace obsolete class

### Testing Instructions
Run `npm run build:css`
Go to System > Language Overrides
Create a new override
Click cursor in Search Text field

### Actual result BEFORE applying this Pull Request
30096-before

### Expected result AFTER applying this Pull Request
30096-after

### CODE REVIEW

**Comment by ceford** [comment]
I don't see any difference, with or without the patch. Can you supply a screen shot showing what to look for?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30096">issues.joomla.org/tracker/joomla-cms/30096</a>.</sub>

**Comment by richard67** [comment]
> I don't see any difference, with or without the patch. Can you supply a screen shot showing what to look for?

Same for me. In both cases without and with this PR I have the popup message above the search box.

**Comment by richard67** [comment]
@ceford I think I found our both mistake: It is not the search box in the toolbar of the list view we have to check, it is the search box at the right bottom place of the edit dialogue when creating a new override or editing an existing one ;-)

**Comment by brianteeman** [comment]
thats why screenshots are so helpful ;)

**Comment by richard67** [comment]
> thats why screenshots are so helpful ;)

Agree.

Beside that, this status message seems to be a tricky thing. Currently it doesn't show up at all, whatever I do.

**Comment by richard67** [comment]
Ahhh .. you have to clear the cache in the list view so the message about cache recreation is shown when entering the search box in the edit view.

**Comment by brianteeman** [comment]
was just typing that info for you ;)

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on ed26f380633ecf6aabd1087f6fcbb5e29270d4bc<br /><br />Very tricky to reproduce the issue.
1. It is the search box at the bottom right corner of the edit dialog when creating a new language override or modifying an existing one, not the one in the list view like I first thought.
2. The status message about cache rebuild only appears if cache cleared in the list view and then logged out and in again, or if using another language and site combination which hasn't been used before.
3. It needs a very large display to reproduce the issue with the message appearing beside the search field and so making it shrink.

But I can confirm that this PR solves that issue.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30096">issues.joomla.org/tracker/joomla-cms/30096</a>.</sub>

**Comment by brianteeman** [comment]
I am fairly certain that this breaks the accessibility by putting the message in a new div instead of inside the input div

**Comment by richard67** [comment]
Even with `aria-hidden="true"` for the message?

**Comment by brianteeman** [comment]
that is on the icon not on the text. You want the text to be accessible and connected to the input

**Comment by richard67** [comment]
I have not tested this item.<br /><br />Changing back to not tested until some feedback from an accessibility expert.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30096">issues.joomla.org/tracker/joomla-cms/30096</a>.</sub>

**Comment by ceford** [comment]
Below the Search Text Field I see a brief message: 'Please Wait while the cache is recreated' on the first click only, with an without the patch. It most vanishes instantly. Clearing the cache in the list view does not make it reappear. It does reappear if I select a different client or language. As far as I can tell the patch does nothing!

**Comment by Quy** [comment]
@ceford What is your monitor's resolution? On 1920x1080, this can be reproduced.

**Comment by Quy** [comment]
Updated with screenshots in the original post.

**Comment by ceford** [comment]
I am using a Macbook Pro - a screen grab is 2880 x 1498 but I think the horizontal resolution is really 1440. I only see the message below the Search Text field so I guess it has already wrapped - mystery solved.

**Comment by richard67** [comment]
@Quy Regarding Brian's comment about accessibility: Would it be an alternative solution to leave HTML markup like it is, i.e. a span, and add some css for block display? This should be still accessible.

**Comment by chmst** [comment]
Not sure if it is a a11y issue. IBut also for sighted users it would be nice to hide the input group while the search is going on.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30096">issues.joomla.org/tracker/joomla-cms/30096</a>.</sub>

**Comment by hans2103** [comment]
@Quy There is no need to put the span `.help-block` in a new div outside div `.input-group`.
It seems that `.input-group` is styled with `display: flex;flex-wrap: wrap;`.
Just add add `flex-basis: 100%` to `.help-block` the issue is solved. 
Can you adjust your PR with this change?

Run `npm build:css` after applying the patch to test the change.

Schermafdruk 2020-07-22 22 34 14

File: build/media_source/com_languages/css/overrider.css [hunk 1]
```diff
@@ -25,6 +25,10 @@
 	display: inline-block;
 }
 
+#refresh-status.help-block {
```
Comment (hans2103) [inline build/media_source/com_languages/css/overrider.css]: for real? Do we really have to work with id's?
That's a pitty. It is consitent with the rest of the document though.

**Comment by hans2103** [comment]
> Just add add `flex-basis: 100%` to `.help-block` the issue is solved.

With reverting your changes the className `.help-block` is gone too. 
Please adjust your PR to a flex-basis on `#refresh-status` only.

I always prevent touching with id's and try to work with classNames instead.
But using id `#refresh-status` is consistent with the rest of the css document.

**Comment by Quy** [comment]
Done. Thanks!

**Comment by hans2103** [comment]
For those who want to test this issue. 
After applying the patch you have to run `npm run build:css`.
After reverting the patch you should run the same command again.

**Comment by hans2103** [comment]
I have tested this item :white_check_mark: successfully on a11e80e0ff21c565859d456408fca86da27cae19<br /><br />run command `npm run build:css` after applying the page to see the change<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30096">issues.joomla.org/tracker/joomla-cms/30096</a>.</sub>

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on a11e80e0ff21c565859d456408fca86da27cae19<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30096">issues.joomla.org/tracker/joomla-cms/30096</a>.</sub>

**Comment by infograf768** [comment]
rtc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30096">issues.joomla.org/tracker/joomla-cms/30096</a>.</sub>

**Comment by infograf768** [comment]
Tks

**Pull request merged**